File: Algoritmo(C)/Alg_S8_2.c
#include <stdio.h>
int main(){
    //variables
    int vetor1[10], vetor2[10], soma[10], i;
    //input
    for(i = 0; i < 10; i++){
        printf("Escreva um valor para o primeiro vetor[%d]: ",i);
        scanf("%d",&vetor1[i]);
        printf("Escreva um valor para o segundo vetor[%d]: ",i);
        scanf("%d",&vetor2[i]);
        //process
        soma[i] = (vetor1[i] + vetor2[i]);
    }
        //output
    for(i = 0; i < 10; i++){
        printf("A soma dos vetores[%d] e : %d\n",i ,soma[i]);
    }
}

File: Algoritmo(Python)/Alg_S8_2.py
#variables
vetor1 = []
vetor2 = []
soma = []
#input + process
for(i) in range(0, 10):
    num1 = int(input("Digite um número para o vetor1 {0}/10: ".format(i + 1)))
    vetor1.append(num1)
    num2 = int(input("Digite um número para o vetor2 {0}/10: ".format(i + 1)))
    vetor2.append(num2)
    soma.append(num1 + num2)
for(j) in (soma):
    print(j)